tests/foreman/longrun/test_inc_updates.py

```diff
@@ -16,279 +16,240 @@
+def qe_lce(module_manifest_org, dev_lce):
+    qe_lce = entities.LifecycleEnvironment(
+        name='QE', prior=dev_lce, organization=module_manifest_org
+    ).create()
+    return qe_lce
+
+
+@pytest.fixture(scope='module')
+def rhel7_sat6tools_repo(module_manifest_org):
+    """Enable Sat tools repository"""
+    rhel7_sat6tools_repo_id = enable_rhrepo_and_fetchid(
+        basearch=DEFAULT_ARCHITECTURE,
+        org_id=module_manifest_org.id,
+        product=PRDS['rhel'],
+        repo=REPOS['rhst7']['name'],
+        reposet=REPOSET['rhst7'],
+        releasever=None,
+    )
+    rhel7_sat6tools_repo = entities.Repository(id=rhel7_sat6tools_repo_id).read()
+    assert rhel7_sat6tools_repo.sync()['result'] == 'success'
+    return rhel7_sat6tools_repo
+
+
+@pytest.fixture(scope='module')
+def custom_repo(module_manifest_org):
+    """Enable custom errata repository"""
+    custom_repo = entities.Repository(
+        url=FAKE_9_YUM_REPO, product=entities.Product(organization=module_manifest_org).create()
+    ).create()
+    assert custom_repo.sync()['result'] == 'success'
+    return custom_repo
+
+
+@pytest.fixture(scope='module')
+def module_cv(module_manifest_org, rhel7_sat6tools_repo, custom_repo):
+    """Publish both repos into module CV"""
+    module_cv = entities.ContentView(
+        organization=module_manifest_org, repository=[rhel7_sat6tools_repo.id, custom_repo.id]
+    ).create()
+    module_cv.publish()
+    module_cv = module_cv.read()
+    return module_cv
+
+
+@pytest.fixture(scope='module')
+def module_ak(module_manifest_org, module_cv, custom_repo, module_lce_library):
+    """Create a module AK in Library LCE"""
+    ak = entities.ActivationKey(
+        content_view=module_cv,
+        environment=module_lce_library,
+        organization=module_manifest_org,
+    ).create()
+    # Fetch available subscriptions
+    subs = entities.Subscription(organization=module_manifest_org).search()
+    assert len(subs) > 0
+    # Add default subscription to activation key
+    sub_found = False
+    for sub in subs:
+        if sub.name == DEFAULT_SUBSCRIPTION_NAME:
+            ak.add_subscriptions(data={'subscription_id': sub.id})
+            sub_found = True
+    assert sub_found
+    # Enable RHEL product content in activation key
+    ak.content_override(
+        data={'content_overrides': [{'content_label': REPOS['rhst7']['id'], 'value': '1'}]}
+    )
+    # Add custom subscription to activation key
+    prod = custom_repo.product.read()
+    custom_sub = entities.Subscription().search(query={'search': f'name={prod.name}'})
+    ak.add_subscriptions(data={'subscription_id': custom_sub[0].id})
+    return ak
+
+
+@pytest.fixture(scope='module')
+def host(
+    module_manifest_org,
+    dev_lce,
+    qe_lce,
+    custom_repo,
+    module_ak,
+    module_cv,
+):
+    # Create client machine and register it to satellite with rhel_7_partial_ak
+    with VMBroker(nick='rhel7', host_classes={'host': ContentHost}) as vm:
```

Comment:
Nice work including the VMBroker migration here!

Reply:
He could also use one of the content host fixtures

Reply:
he would have to add one first